src/main/resources/META-INF/plugin.xml

```diff
@@ -103,6 +108,12 @@
                 <!--                <override-text place="GitMacheteToolbar"/>-->
             </action>
 
+            <action id="GitMachete.MergeCurrentBranchIntoParentAction"
+                    class="com.virtuslab.gitmachete.frontend.actions.toolbar.MergeCurrentBranchIntoParentAction"
+                    icon="AllIcons.Vcs.Merge">
+                <!--                <override-text place="GitMacheteToolbar"/>-->
+            </action>
+
             <action id="GitMachete.OpenMacheteFileAction"
```

Comment:
Hmm is there any reason for `edit` action being specifically b/w `merge` and `slide-out`? likely the actions should be rearranged (here & in context menu)

Reply:
_Slack discussion summary_
Target: 
![image](https://user-images.githubusercontent.com/19799111/83736050-6a0cb180-a651-11ea-84fc-f8e16a9768d8.png)